### PR TITLE
Be able to launch unit tests on multiple versions of PrestaShop 1.7

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Basically,

this is how you can use travis to validate a module: note this is experimental.

Use it at your own risks!

### CODE REVIEW

File: tools/install_module.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Move Module Contents to Install Folder
+echo Move Module Contents to Prestashop Modules Directory
+mkdir     $TRAVIS_BUILD_DIR/modules/$1
+cp -Rf    $MODULE_DIR/*              $TRAVIS_BUILD_DIR/modules/$1/      
+
+# Change Default Language Code to ISO format
+mysql -D prestashop -e "UPDATE ps_lang SET language_code = 'fr-fr' WHERE ps_lang.language_code = 'fr';"
+
+# Enable the Module
+mysql -D prestashop -e "INSERT INTO ps_module ( name ,  active ,  version ) VALUES ('coinbase', 1 , 'test');"
```
Comment (Quetzacoalt91) [inline tools/install_module.sh]: What about using the console?

```
php bin/console prestashop:module install coinbase
```

File: tools/install_prestashop.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+echo Clone Prestashop
+   
+# Clone Prestashop into Build Folder
+cd $TRAVIS_BUILD_DIR
+git clone --depth=1 --branch=$PS_VERSION https://github.com/Prestashop/Prestashop.git $TRAVIS_BUILD_DIR 
+
+# PrestaShop configuration
+cp tests/parameters.yml.travis app/config/parameters.yml
```
Comment (matks) [inline tools/install_prestashop.sh]: This is enough 🤔 ? No need to perform any more steps ? 😮

File: tools/install_module.sh (hunk 1 above)
Comment (eternoendless) [reply tools/install_module.sh]: Good idea, but that command was introduced in which version now?

**Comment by eternoendless** [comment]
How about publishing the install/setup scripts as a separate package that could be imported by modules using composer?

File: tools/install_module.sh (hunk 1 above)
Comment (Quetzacoalt91) [reply tools/install_module.sh]: PrestaShop 1.7.2.0: https://devdocs.prestashop.com/1.7/modules/testing/#module-management-in-command-line

File: tools/install_prestashop.sh (hunk 2 above)
Comment (mickaelandrieu) [reply tools/install_prestashop.sh]: yes!

nahhh, look at https://github.com/mickaelandrieu/coinbase-commerce-prestashop/pull/1/files#diff-d7949b08658567f93cfee17ea671b996R5

Comment (mickaelandrieu) [reply tools/install_prestashop.sh]: I've renamed the script

**Comment by mickaelandrieu** [comment]
> How about publishing the install/setup scripts as a separate package that could be imported by modules using composer?

great spirits meet here :) what I want to be able to do is:

```
composer require prestashop/module-utils
```

and this package will be a Flex recipe adding the scripts to help to test but maybe later we can imagine that this will add commands to help people be more productive when working on the modules.

**Pull request merged**